charts/athens-proxy/templates/deployment.yaml

```diff
@@ -235,15 +239,14 @@ spec:
           subPath: {{ $server.existingSecret.subPath | quote }}
         {{- end }}
         {{- end }}
-        {{- if .Values.image.runAsNonRoot }}
+        {{- with .Values.initContainerSecurityContext }}
```

Comment:
should this point to `securityContext` instead of `initContainerSecurityContext`?

Reply:
yes, updated this